### ISSUE TITLE
Where to next with dashboards?

### ISSUE THREAD

**Issue by parente** [jupyter-incubator/dashboards_server#319]
@parente will fill this in tonight ...

**Comment by rserbitar** [comment]
@parente, thanks for this effort.

I just want to chime in and say that jupyter_dashboards_server along with the layout extension and bundler, provides exactly what is needed. The ability to deploy interactive dashboards with a click of a button to people who dont have jupyter installed locally and do not want to see the code, in a way that looks exactly like the notebook (with the additional extra of getting the ability to do some layouting).

What we basically need is to show others the content of a notebook (including interactivity) without the code. dashboard_servers does this perfectly, and I am concerned to see it get so little attention in the last 8 month (especially after promoting it in the IBM blog first).

**Comment by akhmerov** [comment]
Just chipping in. Interaction with https://github.com/takluyver/nbopen is probably relevant to this issue.

**Comment by cbcunc** [comment]
Good discussion at SciPy about this. More of the Jupyter team starting to understand that it's not just about dashboards for interactive exploration, but also scientists deploying dashboard apps.

**Comment by jasongrout** [comment]
Yes, one-click deploying of dashboard apps has been in discussion for a long time.

**Comment by tylere** [comment]
@cbcunc would you be able to provide a summary of the SciPy discussions on deployable dashboards?
